aws/ae-environment.yml

```diff
@@ -0,0 +1,99 @@
+Parameters:
+    InstanceType:
+        Type: String
+        Description: Instace size
+    AmiId:
+        Type: String
+        Description: AMI id
+    EnvName:
+        Type: String
+        Description: Environment name
+    ApiPort:
+        Type: Number
+        Description: Port number of epoch node API
+        Default: 3013
+    NodeCount:
+        Type: Number
+        Description: Number of epoch nodes
+        Default: 2
+
+Resources:
+
+    ManagementSecurityGroup:
+      Type: 'AWS::EC2::SecurityGroup'
+      Properties:
+          GroupName:
+              Fn::Sub:
+                - "ae-${EnvName}-management"
+                - { EnvName: !Ref EnvName}
+          GroupDescription: Allow node management from internet (ICMP, SSH, API)
+          SecurityGroupIngress:
+          - FromPort: !Ref ApiPort
+            ToPort: !Ref ApiPort
+            CidrIp: 0.0.0.0/0
+            IpProtocol: "tcp"
+          - FromPort: 22
+            ToPort: 22
```

Comment:
I see this file largely similar to https://github.com/aeternity/infrastructure/blob/56828f7de3b64a7004069ddefdd05b8d53bb726b/openstack/ae-environment.yml#L57-L63
Does it mean eventually that one will be removed? Or are them meant to be maintained in parallel?

Reply:
both need to me maintained

Reply:
@lucafavatella We'll deploy both on OS and AWS, they have different orchestration technologies, thus the different files.